config.schema.json

```diff
@@ -253,6 +253,42 @@
             }
           }
         }
+      },
+      "dzk-bacnet": {
```

Comment:
This is used in homebridge ui. With the current configuration the output is always at minimum some JSON-Object. I think the user should be able to remove some item completely or at least an empty object or array. 

```JSON
{
    "name": "BAChome Dynamic Platform",
    "dzk-bacnet": {
        "zones": [
            {
                "zone": 6
            }
        ]
    },
    "platform": "bachome"
}
```

<img width="751" alt="Screenshot of the homebrige ui config screen" src="https://user-images.githubusercontent.com/6471370/204094190-c11419cd-c26f-4d9d-9171-679b0396f7c2.png">